packages/grafana-ui/src/components/MatchersUI/utils.ts

```diff
@@ -79,9 +89,23 @@ export function useSelectOptions(
       if (!found && name === currentName) {
         found = true;
       }
+      const field = displayNames.fields.get(name);
+
+      let previewValues = '';
+      if (field?.values.length && field.values.length > 2) {
+        console.log(fieldTypeIcons[field.type]);
+        previewValues += field.values.get(0);
+        previewValues += `, ${field.values.get(1)}`;
+        previewValues += `, ${field.values.get(2)}`;
+        if (field.values.length > 3) {
+          previewValues += '...';
+        }
+      }
       options.push({
         value: name,
         label: name,
+        description: field ? (previewValues.length > 0 ? `Values: ${previewValues}` : field.type) : undefined,
```

Comment:
lets leave the description/values out for this round -- having the Icon will be a big step forward